src/common/appId.js

```diff
@@ -1,21 +1,19 @@
 import { parseQueryString } from 'binary-common-utils/lib/tools';
-import { addTokenIfValid } from 'binary-common-utils/lib/account';
-import { set as setStorage } from 'binary-common-utils/lib/storageManager';
+import { LiveApi } from 'binary-live-api';
+import {
+    addToken,
+    removeToken,
+    getTokenList,
+    removeAllTokens,
+    get as getStorage,
+    set as setStorage,
+} from 'binary-common-utils/lib/storageManager';
+import { getLanguage } from './lang';
 
 export const setAppId = () => {
```

Comment:
I think this function can be renamed to `setDefaultAppId` to better reflect its functionality.

Reply:
Done